gis/qgis3/Portfile

```diff
@@ -7,10 +7,10 @@ PortGroup           cxx11   1.1
 PortGroup           github  1.0
 PortGroup           qt5     1.0
 
-github.setup        qgis QGIS 3_0_3 final-
-#github.setup        qgis QGIS c625f95
-#github.commit       c625f95
-#revision            20180410
+github.setup        qgis QGIS 3_2_0 final-
+#github.setup        qgis QGIS bc43194
+#github.commit       bc43194
+#revision            20180622
```

Comment:
I'm not sure why these commented out fields are here, but they would seem to be things we should be removing rather than updating...

Reply:
PR is now modified to remove those commented lines!

Reply:
Cool. We may have to wait a bit for @Veence to approve the update since this isn't set openmaintainer.